numcodecs/compat.py

```diff
@@ -50,7 +50,7 @@ def ensure_ndarray(buf):
     return arr
 
 
-def ensure_contiguous_ndarray(buf, max_buffer_size=None):
+def ensure_contiguous_ndarray(buf, max_buffer_size=None, flat=True):
```

Comment:
Preference for using a verb to describe the action taken or not

```suggestion
def ensure_contiguous_ndarray(buf, max_buffer_size=None, flatten=True):
```